.gitpod.Dockerfile

```diff
@@ -1,7 +1,8 @@
 FROM gitpod/workspace-python
-ENV PYTHONUSERBASE=/workspace/.pip-modules
-ENV PATH=$PYTHONUSERBASE/bin:$PATH
-ENV PIP_USER=yes
-
-COPY .gitpod/aliases.txt .
-RUN cat aliases.txt >> $HOME/.bashrc
+USER root
+RUN apt-get update \
+    && apt-get install -y libboost-dev \
+    && apt-get clean && rm -rf /var/cache/apt/* && rm -rf /var/lib/apt/lists/* && rm -rf /tmp/*
+USER gitpod
+RUN pyenv install 3.9.7
+RUN pyenv global 3.9.7
```

Comment:
It's not a big deal, but some of your file edits are missing newlines at the end of the file. Can you change `Files: Insert Final Newline` in the settings UI (`files.insertFinalNewline` in JSON) to `true`.

Reply:
This change was already merged in a PR that was included here. I'll still update it here.